pkg/gopass/secrets/akv_test.go

```diff
@@ -121,6 +121,24 @@ func TestSetKeyValuePairToEmptyAKV(t *testing.T) {
 	assert.Equal(t, "bar", v)
 }
 
+func TestAKVTrailingWhitespace(t *testing.T) {
+	t.Parallel()
+	// Expected behaviour is KEY: VALUE, with one space.
+	// Fallback should exist for KEY:VALUE, with no spaces.
+	mlValue := `foobar
```

Comment:
I think the issue initially was about passwords surrounded by spaces, no just key-values pairs, no?
How does that work for passwords now? A test would be nice.

Reply:
Correct, I think the issue was about whitespace in passwords.
Being able to handle them in KV pairs is very nice as well, but either we need to change the description of the PR to match the implementation or add support for whitespace in passwords as well.

Reply:
I had a misunderstanding between KV pairs and the password itself.  But it should also be simple to not `Trim` the password. But I also see no harm in leaving values trimmed as well.  Do you mind if I do both on the same PR?